### DISCUSSION TITLE
how to use kubebuilder:webhook to gen MutatingWebhookConfiguration namespaceSelector config

### DISCUSSION THREAD

**Discussion by Ghostbaby** [kubernetes-sigs/kubebuilder#2538] [Support]
hi, 
how to use `+kubebuilder:webhook` to gen MutatingWebhookConfiguration namespaceSelector config?

**Comment by camilamacedo86** [comment]
Hello Ghostbaby,

The +kubebuilder:webhook marker is indeed used to generate webhook configurations.  However, it has not an option for namespaceSelector see: https://github.com/kubernetes-sigs/kubebuilder/discussions/2461

Could you please raise it in the controller-tools (https://github.com/kubernetes-sigs/controller-tools) regards this one? It is the project responsible for the markers.
